lib/generators/active_fedora/config/fedora/templates/fcrepo_wrapper_test.yml

```diff
@@ -0,0 +1,4 @@
+#config/fcrepo_wrapper_test.yml.sample
+port: 8986
+enable_jms: false
+fcrepo_home_dir: tmp/fcrepo4-test-data
```

Comment:
I wasn't going to say anything, but if you're fixing @mjgiarlo's things, could we please add a trailing newline to these yml files?